packages/apollo-link-dedup/README.md

```diff
@@ -0,0 +1,51 @@
+# Dedup Link
+
+## Purpose
+An Apollo Link to deduplicate matching requests before making a request.
+
+*NOTE* This link is included by default when using apollo-client.
+
+## Installation
+
+`npm install apollo-link-dedup --save`
+
+To use this package in a web browser or mobile app, you'll need a build system capable of loading NPM packages on the client.

```

Comment:
I don't know if we need this sentence anymore, since more or less everyone is using a build system like this now.